### PR TITLE
add tools images to setting

### PR DESCRIPTION
**Pull request by aiwantaozi**
add logging image to system setting

### CODE REVIEW

File: apis/management.cattle.io/v3/k8s_defaults.go [hunk 1]
```diff
@@ -45,6 +45,11 @@ var (
 		PodInfraContainer:         "rancher/pause-amd64:3.0",
 		Ingress:                   "rancher/nginx-ingress-controller:0.10.2",
 		IngressBackend:            "rancher/nginx-ingress-controller-defaultbackend:1.4",
+		Fluentd:                   "rancher/fluentd:v0.1.4",
+		FluentdHelper:             "rancher/fluentd-helper:v0.1.1",
+		Elaticsearch:              "rancher/docker-elasticsearch-kubernetes:5.6.2",
+		Kibana:                    "kibana:5.6.4",
+		Busybox:                   "busybox",
```
Comment (zionwu) [inline apis/management.cattle.io/v3/k8s_defaults.go]: These images do not depend on k8s version as they do not make API call to k8s.

**Comment by zionwu** [comment]
LGTM

**Comment by gitlawr** [comment]
LGTM

**Comment by alena1108** [comment]
LGTM

**Comment by zionwu** [comment]
related issue: #11367

**Pull request merged**